<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>891</width>
    <height>611</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>播放速度</string>
        </property>
        <widget class="QScrollBar" name="verticalScrollBar">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>30</y>
           <width>16</width>
           <height>160</height>
          </rect>
         </property>
         <property name="value">
          <number>50</number>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>当前时间</string>
        </property>
        <widget class="QLCDNumber" name="lcdNumber">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>50</y>
           <width>88</width>
           <height>165</height>
          </rect>
         </property>
         <property name="digitCount">
          <number>3</number>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="board" name="DMFBboard" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>891</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDMFB">
    <property name="title">
     <string>DMFB设置</string>
    </property>
    <addaction name="menuRowColumnNum"/>
    <addaction name="menuInOutpos"/>
   </widget>
   <widget class="QMenu" name="menuCommand">
    <property name="title">
     <string>命令设置</string>
    </property>
    <addaction name="menuCommadFromFile"/>
   </widget>
   <widget class="QMenu" name="menuWash">
    <property name="title">
     <string>其他设置</string>
    </property>
    <addaction name="menuWash2"/>
   </widget>
   <addaction name="menuDMFB"/>
   <addaction name="menuCommand"/>
   <addaction name="menuWash"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionDefaultPlay"/>
   <addaction name="actionPause"/>
   <addaction name="actionNext"/>
   <addaction name="actionPrevious"/>
   <addaction name="actionReset"/>
   <addaction name="actionRestart"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="menuRowColumnNum">
   <property name="text">
    <string>行列</string>
   </property>
  </action>
  <action name="menuInOutpos">
   <property name="text">
    <string>输入/输出端口</string>
   </property>
  </action>
  <action name="menuCommadFromFile">
   <property name="text">
    <string>文件位置</string>
   </property>
  </action>
  <action name="menuInputFromWindow">
   <property name="text">
    <string>从窗口</string>
   </property>
  </action>
  <action name="menuWash2">
   <property name="text">
    <string>清洗</string>
   </property>
  </action>
  <action name="menuWashAdvanced">
   <property name="text">
    <string>高级设置</string>
   </property>
  </action>
  <action name="actionDefaultPlay">
   <property name="text">
    <string>全部播放</string>
   </property>
  </action>
  <action name="actionNext">
   <property name="text">
    <string>下一步</string>
   </property>
  </action>
  <action name="actionPrevious">
   <property name="text">
    <string>上一步</string>
   </property>
  </action>
  <action name="actionReset">
   <property name="text">
    <string>复位</string>
   </property>
  </action>
  <action name="actionPause">
   <property name="text">
    <string>暂停</string>
   </property>
  </action>
  <action name="actionRestart">
   <property name="text">
    <string>重置设置</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>board</class>
   <extends>QWidget</extends>
   <header>board.h</header>
   <container>1</container>
   <slots>
    <slot>changeSpeed(int)</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>verticalScrollBar</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>DMFBboard</receiver>
   <slot>changeSpeed(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>54</x>
     <y>165</y>
    </hint>
    <hint type="destinationlabel">
     <x>206</x>
     <y>181</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
